projects/collection-editor-library/src/lib/components/meta-form/meta-form.component.ts

```diff
@@ -32,11 +32,12 @@ export class MetaFormComponent implements OnChanges, OnDestroy {
   public previousShuffleValue: boolean;
   public subscription: Subscription;
   public formSectionProperties:any;
+  public formCriteriaSection:any[]=[];
+  public formCriteriaList:any[]= [];
   isDataValidated:boolean= false
```

Comment:
@sohailamjad12 Rename isDataValidated variable to more relevant and generic name-> isValidQuestionCriteria